docs/assets/includes/default-git-provider-settings-tip.md

```diff
@@ -0,0 +1,2 @@
+!!! tip
+    Configure the [default settings](../../organizations/default-git-provider-integration.md) for the integration with your Git provider to ensure that Codacy uses the same settings when enabling the Git provider integration for newly added repositories.
```

Comment:
I feel like this sentence is a bit hard to understand.

What do you think of this alternative?

```suggestion
    Configure the [default Git provider integration settings](../../organizations/default-git-provider-integration.md) that Codacy applies to new repositories to help ensure that all new repositories have the same settings.
```

Reply:
Yep, much simpler! 🤓 👍